src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,165 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+
+    class Node<T> {
+        private T element;
+        private Node<T> next;
+        private Node<T> prev;
+
+        public Node(T element) {
```

Comment:
```suggestion
        Node(T element) {
```
A private inner static class is intended to be used only within its containing class and is not directly accessible from the outside. Since it is meant to be used internally, there is typically no need to declare a public constructor